packages/binding-http/test/http-client-oauth-tests.ts

```diff
@@ -75,26 +76,26 @@ class HttpClientOAuthTest {
     }
 
     @test async "should authorize client with client_credentials flow"() {
-        const scheme: OAuth2SecurityScheme = {
+        const scheme: TD.OAuth2SecurityScheme = {
             scheme: "oauth2",
             flow: "client_credentials",
             token: "https://localhost:3000/token",
             scopes: ["test"],
         };
-        this.client.setSecurity([scheme], { clientId: "thom", clientSecret: "nightworld" });
+        this.client.setSecurity([scheme as SecurityScheme], { clientId: "thom", clientSecret: "nightworld" });
         return this.client.readResource({
             href: "https://localhost:3000/resource",
         });
     }
 
     @test async "should authorize client with resource owener flow"() {
-        const scheme: OAuth2SecurityScheme = {
+        const scheme: TD.OAuth2SecurityScheme = {
```

Comment:
Could we make TD.OAuth2SecurityScheme extending SecurityScheme to avoid casting ?

Reply:
w.r.t. security schemes I still want to wait till a solution for https://github.com/w3c/wot-thing-description/issues/1235 has been landed. At the moment we need to redefine all security schemes. I rather re-use the types defines in TD schema.

Reply:
btw w3c/wot-thing-description#1235 is not resolved 🎉 . Just an heads up 😃